### DISCUSSION TITLE
Please help with installation.

### DISCUSSION THREAD

**Discussion by Jackal28** [pschatzmann/pico-arduino#19] [Q&A] [answered]
I am still new to all of this. I am trying to understand linux as well which will probably help me in the future. Currently I am on a windows machine. I have installed the pico sdk and I utilize arduino IDE, which, even after installing the sdk, does still not include all of the examples in the pico sdk folder. 

Question:
"First you need to install the Pico C Framwork. Please follow the instructions and do not forget to define the PICO_SDK_PATH environment variable."
Could you please provide an example of how to do this?

Also, while I can find multiple pico sdk paths, i cannot locate a single PICO_SDK_ARDUINO_PATH, could you please elaborate?

I would really like to utilize the examples provided within your pico-arduino in the Arduino IDE libraries if that is at all possible but I have no ideal how to install said libraries into Windows. 

Thank you for any advice you could offer as I have spent hours trying to figure this out.

**Comment by pschatzmann** [comment, marked as answer]
If you are a beginner I suggest not to use my framwork and that you stick with the Arduino implementation provided by Earl Phil Hower: https://github.com/earlephilhower/arduino-pico. This one is also based on the development framework provided by Raspberry.

Should you really want to get into the world of c++ development and cmake, here is the link to the official documentation https://datasheets.raspberrypi.org/pico/getting-started-with-pico.pdf. My framework is just added on top of this.
I have installed the pico sdk in the /Users/pschatzmann/Documents/Projects/cpp directory so the setting is
```
export PICO_SDK_PATH=/Users/pschatzmann/Documents/Projects/cpp/pico-sdk
```

I do not recommend to use my examples in the Arduino IDE because of the differences:  In some simple cases it might just work if you rename the extension from cpp to ino (if the example only uses the Arduino API).

**Reply by Jackal28** [reply]
Thank you for your reply, I would like to transition over to C++, but I'm not to sure I'm there yet. Anyways, I do appreciate your quick reply.
